Packages/com.unity.inputsystem/InputSystem/Editor/ProjectWideActions/ProjectWideActionsAsset.cs

```diff
@@ -10,7 +10,7 @@ namespace UnityEngine.InputSystem.Editor
 {
     internal static class ProjectWideActionsAsset
     {
-        internal const string kDefaultAssetPath = "Packages/com.unity.inputsystem/InputSystem/Editor/ProjectWideActions/ProjectWideActionsTemplate.inputactions";
+        internal const string kDefaultAssetPath = "Packages/com.unity.inputsystem/InputSystem/Editor/ProjectWideActions/ProjectWideActions.json";
```

Comment:
No longer a template?

Reply:
Agreed - Would suggest "Template" should still be in the name to make it clear

Reply:
(File still contains this in it with the Template in the name : "name": "ProjectWideActionsTemplate" so keeping name to that with .json appended would make sense)

Reply:
Changed 👍